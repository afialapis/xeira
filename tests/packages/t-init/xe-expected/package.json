{
  "name": "t-init",
  "scripts": {
    "clean-init": "rm -fr xe && rm -fr xeira.json && cp -f package.json.original package.json",
    "x-reset": "npm run --silent clean-init && rm -fr node_modules && rm -fr package-lock.json",
    "make_link": "mkdir -p node_modules && cd node_modules && rm -fr xeira && ln -fs ../../../../ xeira",
    "x-prepare": "npm run --silent x-reset && npm i && npm run --silent make_link",
    "x-init-manual": "node ../../../bin/xeira.mjs init",
    "x-init": "npm run --silent clean-init && mkdir xe && node ../../../bin/xeira.mjs init package both ./src/index.js true false eslint ./lib babel ./dist rollup ./test auto rollup && cp xeira.json ./xe/xeira.json && cp package.json ./xe/package.json",
    "x-compare": "node ../../../src/utils-dev/diff.mjs xe xe-expected",
    "x-run": "npm run --silent x-init && npm run --silent x-compare"
  },
  "main": "./lib/index.cjs",
  "exports": {
    "import": "./dist/t-init.mjs",
    "default": "./lib/index.cjs",
    "require": "./dist/t-init.cjs"
  },
  "module": "./dist/t-init.mjs",
  "cjs": "./dist/t-init.cjs",
  "browser": "./dist/t-init.umd.js",
  "eslintConfig": {
    "extends": [
      "./node_modules/xeira/configs/eslint.react.cjs"
    ]
  },
  "babel": {
    "extends": "./node_modules/xeira/configs/babel.react.cjs"
  }
}
