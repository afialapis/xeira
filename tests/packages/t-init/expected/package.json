{
  "name": "t-init",
  "scripts": {
    "clean-init": "rm -fr xeira.json && cp -f package.json.original package.json",
    "x-reset": "npm run --silent clean-init && rm -fr node_modules && rm -fr package-lock.json",
    "make_link": "mkdir -p node_modules && cd node_modules && rm -fr xeira && ln -fs ../../../../ xeira",
    "x-prepare": "npm run x-reset && npm i && npm run --silent make_link",
    "x-init": "npm run --silent clean-init && node ../../../bin/xeira.mjs init",
    "x-run": "npm run --silent x-init"
  },
  "main": "./dist/t-init.umd.main.js",
  "exports": {
    "import": "./dist/t-init.mjs",
    "default": "./dist/t-init.umd.main.js",
    "require": "./dist/t-init.cjs"
  },
  "module": "./dist/t-init.mjs",
  "type": "module",
  "cjs": "./dist/t-init.cjs",
  "browser": "./dist/t-init.umd.js",
  "eslintConfig": {
    "extends": [
      "./node_modules/xeira/configs/eslint.react.cjs"
    ]
  },
  "babel": {
    "extends": "./node_modules/xeira/configs/babel.react.cjs"
  }
}
